docs/manual/manual_throttle.rst

```diff
@@ -0,0 +1,346 @@
+module to use when interpreting throttle commands. 
+
+A 100% throttle command will cause the module to try to spin at the maximum value for that mode. For example, if the module is in Voltage mode, and its
+maximum voltage is set to 24V, a 100% throttle command will cause the module to apply a drive voltage of 24V. 
+
+Any throttle command less than 100% will be scaled between 0 and the maximum. If the module is in Voltage mode, and its maximum voltage is set to 24V 
+then a 50% throttle command will cause the module to apply a drive voltage of 12V.
+
+Maximum Voltage
+################
+The maximum voltage is used when the module is configured to use :ref:`throttle_voltage_mode` mode. The maximum voltage can be configured using the *Max Volts* parameter under 
+the Tuning tab in IQ Control Center, as shown below.
+
+.. figure:: ../_static/manual_images/throttle/throttle_max_volts.png
+    :align: center
+    :width: 60%
+    :alt: Max Volts Parameter
+
+    Max Volts Parameter in IQ Control Center
+
+Maximum Velocity
+#################
+The maximum velocity is used when the module is configured to use :ref:`throttle_velocity_mode` mode. The maximum velocity can be configured using the *Max Velocity* parameter under the Tuning 
+tab in IQ Control Center, as shown below.
+
+.. figure:: ../_static/manual_images/throttle/throttle_max_velocity.png
+    :align: center
+    :width: 60%
+    :alt: Max Velocity Parameter
+
+    Max Velocity Parameter in IQ Control Center
+
+.. _throttle_direction:
+
+Direction
+**********
+Another important configuration to consider before controlling a Vertiq module is the module’s spin direction. Throttle commands can be from -100% to 100%, 
+and the sign indicates the intended spin direction. Users can configure what direction a positive throttle command should be interpreted as by configuring 
+the direction of the module.
+
+The direction can be clockwise or counterclockwise. When the direction is clockwise, the module will spin clockwise when it receives a positive throttle 
+command and counterclockwise when it receives a negative throttle command. When the direction is counterclockwise, the module will spin counterclockwise 
+when it receives a positive throttle command and clockwise when it receives a negative throttle command.
+
+For example, if the module’s direction is set to clockwise and a 50% throttle command is received, the module will spin at 50% of its maximum value in 
+the clockwise direction. If a -50% throttle command is received, the module will spin at 50% of its maximum value in the counterclockwise direction.
+
+The direction can be set using the *Motor Direction* parameter in the General tab of the IQ Control Center as shown below. Note that this parameter specifies 
+whether the module is 2D or 3D as well as if the direction is clockwise or counterclockwise. The 2D and 3D selection is not important for determining what 

```

Comment:
The 2D and 3D distinction does not determine what direction the motor will spin when receiving a positive throttle command. This is controlled exclusively by the clockwise or counterclockwise designation.